plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -170,6 +175,11 @@ def get_timeout(cls, conn):
         credentials = conn.credentials
         return credentials.timeout_seconds
 
+    @classmethod
+    def get_retries(cls, conn):
+        credentials = conn['credentials']
+        return credentials.get('retries', cls.RETRIES)
```

Comment:
I think you'll want to change this to just be something like
`return conn.credentials.retries`. You can get rid of `RETRIES` then, too.

Reply:
Mirrored the syntax for get_timeout since this does a very similar thing.